NICE.Identity.Management/ClientApp/src/components/App/App.tsx

```diff
@@ -54,6 +58,19 @@ export class App extends React.Component {
 						component={SelectRoles}
 					/>
 					<Route path="/services" exact component={ServicesList} />
+					<Route path="/organisations" exact component={OrganisationsList} />
+					<Route path="/organisations/add" exact component={AddOrganisation} />
+					<Route path="/organisations/:id" exact component={Organisation} />
+					{/* <Route
+						path={"/organisations/:id"}
+						render={({ location }) => {
+							// to stop rendering of component for 'add' route
+							const idRegExp = new RegExp(/[0-9]+$/g);
+							const endOfRoute = location.pathname.split("/").pop() ?? "";
+							return idRegExp.test(endOfRoute) && <Organisation />;
+						}}
+					/> */}
```

Comment:
I found the proper solution to the issue that we were experiencing with the commented out route stuff. Could we replace with the following:

```suggestion
					<Route
						path={"/organisations/:id"}
						render={(props) => {
							// to stop rendering of component for 'add' route
							const idRegExp = new RegExp(/[0-9]+$/g);
							const endOfRoute = props.location.pathname.split("/").pop() ?? "";
							return idRegExp.test(endOfRoute) && <Organisation {...props} />;
						}}
					/>
```